UKHomeOffice/sgmr-service
# Add submit button and submitted page

Needs add people to be merged first

TO TEST
- submit a voyage report
> you should see the submitted confirmation page
> voyage report should be shown under 'submitted' tab on /reports
